clients/python/dd_image_client.py

```diff
@@ -0,0 +1,316 @@
+import base64
+import time
+import json
+import os
+from dd_client import DD
+import cStringIO
+from PIL import Image
+from PIL import ExifTags
+
+
+from scipy import spatial
+
+
+class DDImage(object):
+
+    mllib = 'caffe'
+
+    def __init__(self, host):
+
+        self.host = host
+        self.sname = ''
+        self.description = ''
+        self.mltype = ''
+        self.extract_layer = ''
```

Comment:
I can extract that in the init_for method which controls this and all other inits for extraction. I dont think this belongs in the constructor.